index.html

```diff
@@ -136,7 +136,7 @@ <h1 id="logo"><a href="index.html">Twenty <span>by HTML5 UP</span></a></h1>
 										<header>

```

Comment:
Since this is a tidy up PR, can you please make sure your editor is set to spaces (not tabs) and use 2-space indentation on all files?